e3nn/util/codegen/_mixin.py

```diff
@@ -62,6 +71,8 @@ def __getstate__(self):
             for fname in self.__codegen__:
                 # Get the module
                 smod = getattr(self, fname)
+                if isinstance(smod, fx.GraphModule):
+                    smod = torch.jit.script(jitable(smod))
```

Comment:
so this is correct in the sense that it does need to be a scriptmodule to save it... but raises the question again of why you need a no-compile option in the first place?

Reply:
(in theory I think you _can_ now pickle a GraphModule; I don't fully remember why we abandoned that but I'm sure there was a good reason... probably has something to do with how broken that was on 1.8)